templates/docs/layouts/brochure.md

```diff
@@ -0,0 +1,194 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Brochure site | Layouts
+---
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-9 col-start-large-4">
+      <h1 class="p-heading--2 u-no-margin--bottom">
+        <strong>
+          Brochure site layouts
+        </strong>
+      </h1>
+      <p class="p-heading--2">
+        The rules we use when designing<br class="u-hide--small"> and building pages with Vanilla
+      </p>
+      <div class="p-strip u-no-padding--bottom">
+        <p class="u-no-margin--bottom">
+          A page is a sequence of sections sandwiched between a header and a footer. To create a successful page, we need to get 3 things right:
+        <p>
+        <ul class="p-list--divided" style="max-width: 40rem">{# TODO: replace with max-width utility when implemented #}
```

Comment:
Do we want to add a `max-width` utility or just update the lists to match the `<p>` behaviour?

Reply:
That's a good question that I didn't want to solve in this PR. I'll open a separate issue and we can discuss there.